scripts/vcpkg_ports/conduit/portfile.cmake

```diff
@@ -30,10 +28,14 @@ vcpkg_configure_cmake(
         -DENABLE_UTILS=OFF
         -DBUILD_SHARED_LIBS=${_is_shared}
         -DHDF5_DIR=${CURRENT_INSTALLED_DIR}
+        -DCONDUIT_INSTALL_CONFIG_DIR="share/conduit"
+        -DCONDUIT_INSTALL_CMAKE_MODULE_DIR="share"
 )
 
 vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake
+                          TARGET_PATH share
+                          TOOLS_PATH tools/conduit)
```

Comment:
@cyrush -- This part ended up being really tricky (especially since each change required a `vcpkg remove` / `vcpkg install` step (similar to debugging spack recipes). The code here works, but doesn't seem ideal/elegant.

Some notes:
* `vcpkg` wants all the cmake installation files to be in `<install_dir>/share/conduit`
* `conduit` has some internal logic to modify the output directory for these files
* Without my above modifications, the previous port was placing them in `share/conduit/conduit`
* The corresponding logic in conduit is here: https://github.com/LLNL/conduit/blob/de9d180acf0f954d50aa0bac20fb97cb5c063362/src/config/CMakeLists.txt#L37-L85

Reply:
Looks like there's something different in my environment from that of the "github actions":
* https://github.com/LLNL/axom/actions/runs/965032062

Axom's config failed there due to bad paths in conduit's config:
```
CMake Error in axom/CMakeLists.txt:
  Imported target "conduit::conduit" includes non-existent path

    "D:/a/axom/axom/uberenv_libs/vcpkg/installed/include/"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.
  * An install or uninstall procedure did not complete successfully.
  * The installation package was faulty and references files it does not provide.
```
See: https://github.com/LLNL/axom/runs/2897044456?check_suite_focus=true#step:8:526

Reply:
this looks just like the issue @white238  and @joshessman-llnl saw when looking for conduit in the old vs new way. 

Yes, the cmake files for conduit are in `lib/cmake/conduit`, not under share. 

Conduit uses relative location of the cmake folder to help understand everything else. Moving it relative to the dir structure in a new way isn't going to work.

I'll have to think about this.

Reply:
Thanks @cyrush  -- I got it working in `vcpkg` with a very small patch (that only `vcpkg` applies) -- see my comment below.